app/script/search/SearchRepository.js

```diff
@@ -76,38 +75,63 @@ z.search.SearchRepository = class SearchRepository {
       return userEntities;
     }
     const SEARCHABLE_FIELDS = z.search.SearchRepository.CONFIG.SEARCHABLE_FIELDS;
```

Comment:
Since you are in the same file you don't need the `z.search` -> `CONFIG.SEARCHABLE_FIELDS` will do

Reply:
Ahhh, good to know :+1: